src/main/java/com/kodilla/ecommercee/GroupController.java

```diff
@@ -1,4 +1,33 @@
 package com.kodilla.ecommercee;
 
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/v1/ecommercee")
 public class GroupController {
+
+    @RequestMapping(method = RequestMethod.GET, value = "allProductGroups")
```

Comment:
Make it more readeable by using @GetMapping instead
```suggestion
    @GetMapping(value = "allProductGroups")
```